test/clj/rems/api/test_applications.clj

```diff
@@ -2186,10 +2193,15 @@
                                :label {:en "disease specific research"}
                                :shorthand "DS"}]
                   :duo/matches [{:resource/id res1
+                                 :duo/id "DUO:0000016"
+                                 :duo/label {:en "genetic studies only"}
+                                 :duo/shorthand "GSO"
+                                 :duo/validation {:errors [] :validity "duo/not-found"}}
+                                {:resource/id res1
                                  :duo/id "DUO:0000024"
                                  :duo/label {:en "publication moratorium"}
                                  :duo/shorthand "MOR"
-                                 :duo/validation {:errors [] :validity "duo/not-found"}}
+                                 :duo/validation {:errors [] :validity "duo/not-compatible"}}
```

Comment:
this API test now validates that `:date` restriction can be submitted with no data, and it should not error.